pytrnsys/rsim/executeTrnsys.py

```diff
@@ -162,6 +162,11 @@ def moveFileFromSource(self):
         logger.debug("Path %s" % self.path)
         logger.debug("PathOutput %s" % self.pathOutput)
 
+        if len(fileSource) >= 260:
```

Comment:
I'd maybe say "exceeding the _Windows_ path length limit" as I might otherwise think this is an arbitrary pytrnsys limit and just delete the code. Also, no need for an "elif" after "raise".